src/v/cluster/members_table.h

```diff
@@ -37,13 +37,16 @@ class members_table {
 
     bool contains(model::node_id) const;
 
-    void update_brokers(const std::vector<model::broker>&);
+    void update_brokers(model::offset, const std::vector<model::broker>&);
 
-    std::error_code apply(decommission_node_cmd);
-    std::error_code apply(recommission_node_cmd);
+    std::error_code apply(model::offset, decommission_node_cmd);
+    std::error_code apply(model::offset, recommission_node_cmd);
+
+    model::offset version() const { return _version; }
```

Comment:
maybe we can change returned type to `model::revision_id` it can be created from underlying offset but abstraction is more clear.

Reply:
done